packages/react/src/hooks/useSendTransaction.ts

```diff
@@ -1,26 +1,30 @@
 import { useMutation } from '@tanstack/react-query';
-import type { TransactionRequestLike } from 'fuels';
+import {
+  Address,
+  type AbstractAddress,
+  type TransactionRequestLike,
+} from 'fuels';
 
 import { useFuel } from '../providers';
 
 type UseSendTransactionParams = {
-  address: string;
+  address: string | AbstractAddress;
   transaction: TransactionRequestLike;
 };
 
 export const useSendTransaction = () => {
   const { fuel } = useFuel();
 
   const { mutate, mutateAsync, ...queryProps } = useMutation({
-    mutationFn: (params: UseSendTransactionParams) => {
-      return fuel.sendTransaction(params.address, params.transaction);
+    mutationFn: ({ address, transaction }: UseSendTransactionParams) => {
+      const destination = Address.fromAddressOrString(address).toString();
```

Comment:
I love the idea of improving the input flexibility and receive the `AbstractAddress` as well, but using `fromAddressOrString` is not safe as the logic inside that method is limited and don't check all types of strings.

better to use:
`Address.fromDynamicInput(fromDynamicInput).toString()`

Reply:
https://github.com/FuelLabs/fuels-npm-packs/pull/97/commits/3c052e2ed514fd78621c6684e3c4fcba242c99d4